templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/service/impl/LoginController.java

```diff
@@ -28,7 +28,7 @@
    * @param logout flag for successful logout
    * @return the view model
    */
-  @RequestMapping(value = "/login**", method = RequestMethod.GET)
+  @RequestMapping(value = "/login**", method = {RequestMethod.GET,RequestMethod.POST})
```

Comment:
improvement that will do no harm.
BTW: You do not seem to use oasp4j-ide as eclipse would format the code on save via save-action and should have added a blank after comma. No worries for this PR but please check your IDE setup for further PRs.